sidebars.js

```diff
@@ -14,18 +14,18 @@
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
+  tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
 
   // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    {
-      type: 'category',
-      label: 'Tutorial',
-      items: ['hello'],
-    },
-  ],
-   */
+  // /*
+  // tutorialSidebar: [
+  //   {
+  //     type: 'category',
+  //     label: 'Tutorial',
+  //     items: ['hello'],
+  //   },
+  // ],
+  //  */
```

Comment:
This can be removed, right?

Reply:
@ferrarafer yes, but none of us know the framework well enough to figure out how to add it back lol

Reply:
@ferrarafer It was already commented by the framework but I double commented it by accident